### PR TITLE
Update LoadTestingGuidelines.md

### PR DESCRIPTION
**Pull request by mitovskaol**

### CODE REVIEW

File: docs/LoadTest/LoadTestingGuidelines.md [hunk 1]
```diff
@@ -49,3 +49,8 @@ Providing a comprehensive test plan that includes all abovementioned  informatio
 Second, you'll need to speak to the Platform Services Team about scheduling your test (for after 5:30pm on a weekday, remember!) - we'll need to take a look at your load test plan, make sure we have all the information we need for the test, and make sure that we have someone available during your chosen testing window. We'll work with you if we need to request any alterations to either the time/date or the plan! Contact [@olena.mitovska](https://chat.developer.gov.bc.ca/direct/olena.mitovska) in RocketChat.
 
 Assuming all goes well, your plan will be green-lit, and... that's it! Off you go to perform your load test!
+
+<em>Love, Platform Services Team, xo</em>
+
+<sub><sup><a name="myfootnote1">1</a>: As of spring 2021, the Silver cluster of OpenShift 4 Platform includes 6 router nodes which together can handle 60K **concurrent active connections**. Browsers and other clients will usually hold a connection open for some time and make multiple requests in the same connection or use a long running connection for things like websockets. To keep the cluster stable and healthy, only 2/3 of the overall routing capacity can be targeted by the application load test.</sup></sub>
```
Comment (StevenBarre) [inline docs/LoadTest/LoadTestingGuidelines.md:55]: Does this need both sub and sup? Shouldn't it be just one or the other?

**Comment by mitovskaol** [comment]
I used Service Defintiion as an example that has footnotes https://raw.githubusercontent.com/BCDevOps/openshift-wiki/master/docs/OCP/OCPServiceDefinition.md

File: docs/LoadTest/LoadTestingGuidelines.md (hunk 1 above)
Comment (mitovskaol) [reply docs/LoadTest/LoadTestingGuidelines.md:55]: The footnotes look good in the Service Definition so I just copied the markdown from there https://raw.githubusercontent.com/BCDevOps/openshift-wiki/master/docs/OCP/OCPServiceDefinition.md

Comment (StevenBarre) [reply docs/LoadTest/LoadTestingGuidelines.md:55]: kk.

**Review by StevenBarre — APPROVED** [review]

**Pull request merged**